scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -371,15 +403,15 @@ class Router(formatOps: FormatOps) {
           case _: Mod => Space
           case t: Term.Name
               if style.spaces.afterTripleEquals &&
-                t.tokens.map(_.syntax) == Seq("===") =>
+                t.tokens.map(_.syntax) == Array("===") =>
```

Comment:
Are you sure this preserves the original semantics? 

```scala
@ Array("==") == Array("==")
res0: Boolean = false
```

Reply:
Use `sameElements` instead
```scala
(t.tokens.map(_.syntax) sameElements Array("==="))
```

Reply:
I guess you are right i will look into this since array is like a primitive type with corner case semantics.